src/lib/pages/schedule-view.svelte

```diff
@@ -260,11 +263,40 @@
           <ScheduleError error={schedule?.info?.invalidScheduleError} />
         </div>
       {/if}
+      {#if $groupByCountEnabled}
+        <div class="flex w-full flex-col gap-2 text-lg">
+          <div class="flex items-center gap-2">
+            <span data-testid="workflow-count"
+              >{$workflowCount.count.toLocaleString()}
+              <Translate
+                key={$workflowCount.count === 1
+                  ? 'common.workflow'
+                  : 'common.workflows'}
+              />
```

Comment:
I18next support pluralization out of the box, un `src/lib/i18n/locales/common.ts` you can have 2 versions of the same key:
```
{
  "workflows_one": "Workflow",
  "workflows_other": "Workflows",
}
```
And then just pass a `count` prop to `<Translate />`, i.e. `<Translate key="common.workflows" count={$workflowCount.count} />`

Reply:
Nice nice nice, I was definitely using count wrong when i tried this.